components/tensorflow/tensorboard/prepare_tensorboard/component.yaml

```diff
@@ -39,7 +35,6 @@ implementation:
           }
       ' >"$output_metadata_path"
     - {inputValue: Log dir URI}
-    - {outputPath: Log dir URI}
```

Comment:
This is a breaking change. This output is supposed to be used by most pipelines that use this component. The trainer's Log dir URI should come from the Prepare Tensorboard component.

Reply:
Yes, we can add the output back once fixing https://github.com/kubeflow/pipelines/issues/5831.